exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
+{[String, String|Error]*} cases => {
+  // empty strand
+  ["",                         ""],
+  // single C nucleotide strand
+  ["C",                       "G"],
+  // single G nucleotide strand
+  ["G",                       "C"],
+  // single A nucleotide strand
+  ["A",                       "U"],
+  // single T nucleotide strand
+  ["T",                       "A"],
+  // longer strand
+  ["ACGTGGTCTTAA", "UGCACCAGAAUU"],
+  // strand with invalid nucleotides
+  ["XCGFGGTDTTAA",        Error()],
```

Comment:
an unfortunate consequence of saying `Error()` here (instead of some other indication that something of type `Error` is expected) is that now I think this makes it impossible to have an `Error` that contains additional information, which students might like to do (for example `Error(InvalidNucleotide('X'))`) ?

Ah, unless the student makes a `InvalidNucleotide` subclass of `Error` and allows the `InvalidNucleotide` to contain the character. If that is possible (I think it is, but I've forgotten), then no objection.